server/schema/article/article.resolver.js

```diff
@@ -266,6 +266,47 @@ module.exports = {
       throw APIError(null, error);
     }
   },
+  listArticlesByYearandMonth: async (
+    _parent,
+    { onlyPublished, limit = DEF_LIMIT, offset = DEF_OFFSET, year, month },
```

Comment:
@120EE0692 optional arguments should always be kept at the end

```suggestion
    { year, month, onlyPublished, limit = DEF_LIMIT, offset = DEF_OFFSET },
```